Formula/transcrypt.rb

```diff
@@ -1,17 +1,20 @@
 class Transcrypt < Formula
   desc "Configure transparent encryption of files in a Git repo"
   homepage "https://github.com/elasticdog/transcrypt"
-  url "https://github.com/elasticdog/transcrypt/archive/v2.2.0.tar.gz"
-  sha256 "6ea1dd8c3c7306e54f056728c7344ed31cf52403566e6c2268acc0f984d53391"
+  url "https://github.com/elasticdog/transcrypt/archive/v2.2.1.tar.gz"
+  sha256 "6453d2b2c3f0d903d65d9658d638f40d6e3aeba29ed792919f21477ea570a525"
   license "MIT"
   head "https://github.com/elasticdog/transcrypt.git", branch: "main"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "c2808cb2a2183c73a48c6588ad8678367dcfa4447eb88d12daeb591e0f37cf22"
   end
 
+  uses_from_macos "vim" => :build # needed for xxd
```

Comment:
since vim is a runtime dep, i think we just want this right (and remove line 17)

```suggestion
  uses_from_macos "vim" # needed for xxd
```

Reply:
`transcrypt` doesn't use `xxd` on macOS. `xxd` is a requirement only for OpenSSL 3.
- macOS currently uses LibreSSL. This could change if we decide to make `openssl@3` non-keg-only #104784 in which case `uses_from_macos "vim"` would be correct.
- Linux situation is a bit tricky as it is using `openssl` from system, which means it is OpenSSL 3 on Ubuntu 22 but could be something else on other systems. Also, if you install Homebrew's `openssl@1.1`, it will be used and then `vim` is not necessary, e.g. #119966

Reply:
I'm not crazy about adding a dependency on OpenSSL 1.1 given how we are only a few months out from EOL. I'll drop the `uses_from_macos` and leave the `vim` dip on Linux.

Reply:
As note, `vim` automatically installs `openssl@1.1`, so `transcrypt` will implicitly use `openssl@1.1`

Though, when we migrate `vim` to `openssl@3` and symlink `openssl@3`, it will implicitly migrate `transcrypt` to use `openssl@3`.

Reply:
Was not aware of that. Thanks for the heads up!